cli/dcoscli/data/help/package.txt

```diff
@@ -5,24 +5,22 @@ Usage:
     dcos package --config-schema
     dcos package --help
     dcos package --info
-    dcos package describe [--app --cli --config]
-                          [--render]
-                          [--package-versions]
-                          [--options=<file>]
+    dcos package describe <package-name>
+                          [--app --cli --config --render --options=<file>]
```

Comment:
should we put `render` and `options` on their own line since they are slightly different then app/cli/config options